package.json

```diff
@@ -22,6 +23,8 @@
     "test:coverage": "API=stubs jest --coverage",
     "test:e2e": "API=stubs playwright test",
     "test:e2e:stubs": "npm run test:e2e",
+    "audit": "npm run clean && playwright test --config=playwright-lighthouse.config.ts",
```

Comment:
Should be renamed as `lighthouse:audit` & `lighthouse:audit:debug` so it's not confused with [npm audit](https://docs.npmjs.com/cli/v9/commands/npm-audit)